test/Microsoft.IdentityModel.Tokens.Tests/CryptoProviderCacheTests.cs

```diff
@@ -44,7 +44,7 @@ public class CryptoProviderCacheTests
     {
         public static Func<CryptoProviderCache> CreateCacheForTesting = new Func<CryptoProviderCache>(() =>
         {
-            return new InMemoryCryptoProviderCache(new CryptoProviderCacheOptions(), TaskCreationOptions.None, 50);
+            return new InMemoryCryptoProviderCache(new CryptoProviderCacheOptions(), 50);
```

Comment:
>50 [](http://example.com/codeflow?start=85&length=2)

Wondering if it's still needed to set this value? This was needed in the first lru implementation but many things have changed since. 

@mafurman thoughts?

Reply:
I commented on this earlier; the tryTakeTimeout is no longer relevant. :)